crates/rnote-ui/src/penssidebar/toolspage.rs

```diff
@@ -15,6 +18,16 @@ mod imp {
         pub(crate) toolstyle_offsetcamera_toggle: TemplateChild<ToggleButton>,
         #[template_child]
         pub(crate) toolstyle_zoom_toggle: TemplateChild<ToggleButton>,
+        #[template_child]
+        pub(crate) verticaltool_menubutton: TemplateChild<MenuButton>,
+        #[template_child]
+        pub(crate) verticaltool_popover: TemplateChild<Popover>,
+        #[template_child]
+        pub(crate) verticaltool_popover_close_button: TemplateChild<Button>,
+        #[template_child]
+        pub(crate) verticalspace_respect_vertical_bordersrow: TemplateChild<adw::SwitchRow>,
+        #[template_child]
+        pub(crate) verticalspace_respect_horizontal_bordersrow: TemplateChild<adw::SwitchRow>,
```

Comment:
I'd rename these fields in the following way: `verticaltool` -> `verticalspace` and respect_<horizontal/vertical>_bordersrow` -> `limit_movement_<horizontal/vertical>_borders_row`